package.json

```diff
@@ -147,7 +147,7 @@
     "rcfile": "1.0.3",
     "read-pkg-up": "7.0.1",
     "replace": "1.1.5",
-    "rollup": "1.32.0",
+    "rollup": "2.0.2",
```

Comment:
The question might be if we want to jump onto it already.

1. Node 10 to run ✅ 
2. Structure of Rollup's ESM builds has changed ✅ 
   - Should be internal only. Either plugins might consume it but we don't
3. watch.chokidar" option no longer accepts a boolean ✅ 
   - Have not seen us using it all too much anyway (watch mode)
4. experimentalOptimizeChunks and chunkGroupingSize options have been removed
   - Have not seen us using it
5. Plugin hooks have been removed
   - Means the ecosystem has to adapt but given we test the bundles

Reply:
I think we can wait a bit for the plugins to catch up.

Reply:
...on the other hand building works just fine. We might also just give it a try?

Reply:
Also true. Well we can try it I guess